docs/README.md

```diff
@@ -0,0 +1,299 @@
+
+> The following guide is adapted from [🤗 Transformers](https://github.com/huggingface/transformers/tree/main/docs).
+
+To generate the documentation for 🤗 Optimum intel, simply run the following
+command from the root of the `optimum-intel` repository:
+
+```bash
+make doc BUILD_DIR=intel-doc-build
+```
+
+This command will generate the HTML files that will be rendered as the
+documentation on the [Hugging Face
+website](https://huggingface.co/docs/optimum/index). You can inspect them in
+your favorite browser. You can also adapt the `BUILD_DIR` argument to any
+temporary folder that you prefer.
+
+---
+**NOTE**
+
+You only need to generate the documentation to inspect it locally, e.g. if you're
+planning changes and want to check how they look like before committing. You
+should not commit the built documentation.
+
+---
+
+# Writing documentation - specification
+
+The 🤗 Optimum Habana documentation follows the [Google
+documentation](https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html)
+style for docstrings, although we can write them directly in Markdown.
+
+## Adding a new element to the navigation bar
+
+Under the hood, the documentation is generated by the
+[`hf-doc-builder`](https://github.com/huggingface/doc-builder) library. Here we
+summarize the main syntax needed to write the documentation -- consult
+`hf-doc-builder` for more details. 
+
+Accepted files are Markdown (.md or .mdx).
+
+Create a file with its extension and put it in the `docs/source` directory. You
+can then link it to the table of contents by putting the filename _without the
+extension_ in the
+[`_toctree.yml`](https://github.com/huggingface/optimum/blob/main/docs/source/_toctree.yml)
+file.
+
+## Renaming section headers and moving sections
+
+It helps to keep the old links working when renaming section header and/or
+moving sections from one document to another. This is because the old links are
+likely to be used in Issues, Forums and social media and it makes for a much
+more superior user experience if users reading those months later could still
+easily navigate to the originally intended information.
+
+Therefore we simply keep a little map of moved sections at the end of the
+document where the original section was. The key is to preserve the original
+anchor.
+
+So if you renamed a section from: "Section A" to "Section B", then you can add
+at the end of the file:
+
+```
+Sections that were moved:
+
+[ <a href="#section-b">Section A</a><a id="section-a"></a> ]
+```
+and of course if you moved it to another file, then:
+
+```
+Sections that were moved:
+
+[ <a href="../new-file#section-b">Section A</a><a id="section-a"></a> ]
+```
+
+Use the relative style to link to the new file so that the versioned docs
+continue to work.
+
+For an example of a rich moved sections set please see the very end of [the
+`Trainer`
+doc](https://github.com/huggingface/transformers/blob/main/docs/source/main_classes/trainer.mdx)
```

Comment:
Current link not working
```suggestion
doc](https://github.com/huggingface/transformers/blob/main/docs/source/en/main_classes/trainer.mdx)
```

Reply:
Good catch! I opted for a slightly different URL in the end